src/OpenSage.Game/Data/W3d/W3dPrelitLightMapMultiTexture.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Numerics;
+using OpenSage.Data.Utilities.Extensions;
+
+namespace OpenSage.Data.W3d
+{
+    public sealed class W3dPrelitLightMapMultiTexture : W3dContainerChunk
```

Comment:
Is it worth making a base class for this and `W3dPrelitLightMapMultiPass` and `W3dPrelitUnlit` and `W3dPrelitVertex`, and moving a lot of stuff into that? Seems like they're basically the same thing?

Reply:
Yes, I had meant to do but somehow got sidetracked! I'll add it in the update.

Reply:
See PrelitBase